plugin/accent.vim

```diff
@@ -63,6 +67,6 @@ set completefunc=AccentCompletion
 nnoremap <silent><script> <Plug>(accent) :call <SID>cycle(matchstr(getline('.'), '\%'.col('.').'c.'))<CR>
 
 if !hasmapto('<Plug>(accent)', 'n')
-  nmap gx <Plug>(accent)
+  nmap tx <Plug>(accent)
```

Comment:
What's the thinking behind this change?

Reply:
"gx"` is the default command to open a url under the cursor, so it breaks that feature. I'll gladly change it to whatever you think works best.

Reply:
sounds good. I'll update the PR.

Reply:
Thanks!